From: Michael Froman <mjfroman@mac.com>
Date: Mon, 18 Nov 2024 17:59:01 -0600
Subject: (tmp-cherry-pick) Revert "Disable LibaomAv1Encoder tests to unblock
 Chromium roll" (084f1ae4bb)

This reverts commit f8b3dab7c6320a9890f0b003b43d7099e2e00a5b.

Reason for revert: The fix landed in libaom (https://aomedia-review.googlesource.com/c/aom/+/193761) and it is now available in WebRTC (import CL: https://webrtc-review.googlesource.com/c/src/+/364126).

Original change's description:
> Disable LibaomAv1Encoder tests to unblock Chromium roll
>
> The tests exercise the new encoder API that is not used in prod yet.
>
> Bug: webrtc:369633254
> Change-Id: Iee6bc16ebd471f4accdd9531cdb404f159557f51
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/363820
> Commit-Queue: Sergey Silkin <ssilkin@webrtc.org>
> Reviewed-by: Rasmus Brandt <brandtr@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#43083}

Bug: webrtc:369633254
Change-Id: Ia02db32f7f09e3abc3d0a46605feeabd82673f06
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/364281
Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
Reviewed-by: Sergey Silkin <ssilkin@webrtc.org>
Reviewed-by: Rasmus Brandt <brandtr@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#43120}
---
 api/video_codecs/libaom_av1_encoder_factory_test.cc | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/api/video_codecs/libaom_av1_encoder_factory_test.cc b/api/video_codecs/libaom_av1_encoder_factory_test.cc
index 594f9fc066..0d87a8c57b 100644
--- a/api/video_codecs/libaom_av1_encoder_factory_test.cc
+++ b/api/video_codecs/libaom_av1_encoder_factory_test.cc
@@ -420,8 +420,8 @@ TEST(LibaomAv1Encoder, InputResolutionSwitching) {
   EXPECT_THAT(Resolution(f2), ResolutionIs(160, 90));
   EXPECT_THAT(Psnr(in2, f2), Gt(40));
 }
-// TODO(webrtc:369633254): Re-enable after the libaom issue is fixed.
-TEST(LibaomAv1Encoder, DISABLED_TempoSpatial) {
+
+TEST(LibaomAv1Encoder, TempoSpatial) {
   auto frame_reader = CreateFrameReader();
   auto enc = LibaomAv1EncoderFactory().CreateEncoder(kCbrEncoderSettings, {});
 
@@ -507,8 +507,7 @@ TEST(DISABLED_LibaomAv1Encoder, InvertedTempoSpatial) {
   EXPECT_THAT(Resolution(dec.Decode(tu2_s1.bitstream)), ResolutionIs(640, 360));
 }
 
-// TODO(webrtc:369633254): Re-enable after the libaom issue is fixed.
-TEST(LibaomAv1Encoder, DISABLED_SkipMidLayer) {
+TEST(LibaomAv1Encoder, SkipMidLayer) {
   auto frame_reader = CreateFrameReader();
   auto enc = LibaomAv1EncoderFactory().CreateEncoder(kCbrEncoderSettings, {});
 
